test/Microsoft.Extensions.Options.Test/OptionsSnapshotTest.cs

```diff
@@ -209,6 +209,85 @@ public void CanConfigureAllDefaultAndNamedOptions()
         }
 
         [Fact]
+        public void CanConfigureAndInitializeAllDefaultAndNamedOptions()
+        {
+            var services = new ServiceCollection().AddOptions();
+            services.ConfigureAll<FakeOptions>(o => o.Message += "Default");
+            services.Configure<FakeOptions>(o => o.Message += "0");
+            services.Configure<FakeOptions>("1", o => o.Message += "1");
+            services.InitializeAll<FakeOptions>(o => o.Message += "Initialize");
+            services.Initialize<FakeOptions>(o => o.Message += "2");
```

Comment:
To clarify, whatever value we pick, its just used as the default for the overloads that don't specify a name, so it doesn't really have any documention value other than if you were debugging a custom IConfigure/InitializeOptions.